apiary.apib

```diff
@@ -197,7 +213,7 @@ Polls is a simple API allowing consumers to view polls and vote in them.
             ]
         }
 
-## Questions Collection [/districts-info?district=[11, 12, 13]&relative=true]
+## Questions Collection [/districts-info?relative=true]
```

Comment:
could we merge this call and /province-info? It could be called area-info and the given ids are just areaIds so districts or provinces?